server/src/models/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from "./errors/apiError";
```

Comment:
Yes, I think this is a common pattern. I have seen this on many angular projects. Yes, we should add this pattern for services, Middleware, etc.